src/victory-util/prop-types.js

```diff
@@ -127,11 +123,12 @@ export const domain = makeChainable((props, propName, componentName) => {
  */
 export const scale = makeChainable((props, propName, componentName) => {
   const supportedScaleStrings = ["linear", "time", "log", "sqrt"];
+  const isFunction = (val) => typeof val === "function";
```

Comment:
Yes, unless we can do a bit of research for `typeof function () {}` behavior and ensure that the `isFunction` defined here works with our browser support matrix (also, do we have a browser support matrix?).